contracts/margin/external/exchangewrappers/ZeroExV2MultiOrderExchangeWrapper.sol

```diff
@@ -205,30 +251,78 @@ contract ZeroExV2MultiOrderExchangeWrapper is
                 );
             }
 
+            // ignore orders that the maker will not be able to fill
+            if (!makerHasEnoughTokens(
```

Comment:
soo, does this basically just ignore partial amounts?

Like it's possible a maker only has enough tokens to fill part of the order. Actually what does 0x even do in that case with their batch fill no throw function? Does it just totally ignore the order, or does it fill up the the amount the maker has?

Reply:
It tries to max fill unless it's going to run up against the maximum that you give it. So this is modeling what batchfillnothrow is going to do